ecosystem/typescript/sdk_v2/src/api/general.ts

```diff
@@ -38,14 +39,30 @@ export class General {
     return chainId;
   }
 
+  /**
+  * Call for a move view function
+  *
+  * @param payload Transaction payload
+  * @param ledger_version (optional) Ledger version to lookup block information for
+  *
+  * @returns MoveValue[]
+  */
+  async view(payload: Gen.ViewRequest, ledger_version?: string): Promise<Gen.MoveValue[]>
+
   /**
    * Call for a move view function
    *
    * @param payload Transaction payload
-   * @param version (optional) Ledger version to lookup block information for
+   * @param ledger_version (optional) Ledger version to lookup block information for
```

Comment:
let's add some more comments on this interface and how one should use it

Reply:
added